README.MD

```diff
@@ -105,6 +105,14 @@ Useful in a situation where the module uses variables or mixins without explicit
 
 For example you have vars in file, but without implicit import. You can prepend file with `sass` variables before convert module.
 
+#### inject
```

Comment:
Let's rename to something like this `injectIntoJSDOM`.

Reply:
Renaming it to `injectIntoDOM` might be better, to cover any non JSDOM env jest might be running in.